tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -456,10 +515,12 @@ handle_build() {
 		install_firecracker
 		install_image
 		install_initrd
+		install_initrd_sev
 		install_kernel
```

Comment:
Should there be an entry for the SEV Kernel here?

Reply:
We don't need entry for kernel because kernel builds here: https://github.com/kata-containers/kata-containers/pull/6591/files#diff-e157865d84d98087a8593904adb644a4d5348f59de4ca7111e9fedb9f406de70R95